### PR TITLE
fix: added the sidebar

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
-  Added the sidebar to the avatar
- made some structural changes.

### CODE REVIEW

File: Documentation/avatars/avatars.html [hunk 1]
```diff
@@ -3,52 +3,153 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <link
-      rel="stylesheet"
-      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
-      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
-      crossorigin="anonymous"
-    />
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
     <link rel="stylesheet" href="/components/avatars.css" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (shuntzhere) [inline Documentation/avatars/avatars.html:8]: Make a base css file and import your components css file to it and use that.

**Review by Sameony — APPROVED** [review]
The code is great, nice

**Review by shuntzhere — COMMENTED** [review]
Suggested a minor change, the HTML looks good. CSS file is missing.

File: Documentation/avatars/avatars.html (hunk 1 above)
Comment (vaibhav-mahalle) [reply Documentation/avatars/avatars.html:8]: will look into it

**Pull request closed without merging**